doc/plutus/tutorials/plutus-tx.rst

```diff
@@ -211,3 +211,18 @@ The output is the encoded version of ``False``.
 .. literalinclude:: BasicPlutusTx.hs
    :start-after: BLOCK10
    :end-before: BLOCK11
+
+Plutus Tx Plugin Options
```

Comment:
These don't belong in a tutorial, they're reference documentation. I think add a page under `Reference` for them.

Reply:
You could include a section here about how to use them, and then link to the reference doc for the enumeration of the options and what they  do.